utilities/unittests/test_oadp.py

```
@@ -75,19 +80,31 @@ def test_delete_velero_resource_pod_not_found(self, mock_get_pod):
         with pytest.raises(AttributeError):
             delete_velero_resource(resource=mock_resource, client=mock_client)
 
+    @pytest.mark.parametrize(
+        "kind,name",
```

Ruff (0.14.10)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)